packages/compass/src/app/components/home.tsx

```diff
@@ -152,21 +147,23 @@ function HomeWithConnections({
   return (
     <ConnectionStorageProvider value={connectionStorage}>
       <FileInputBackendProvider createFileInputBackend={createFileInputBackend}>
-        <CompassConnections
-          appName={props.appName}
-          onExtraConnectionDataRequest={getExtraConnectionData}
-          onAutoconnectInfoRequest={onAutoconnectInfoRequest}
-          doNotReconnectDisconnectedAutoconnectInfo
-          onFailToLoadConnections={(error) => {
-            openToast('failed-to-load-connections', {
-              title: 'Failed to load connections',
-              description: error.message,
-              variant: 'warning',
-            });
-          }}
-        >
-          <Home {...props}></Home>
-        </CompassConnections>
+        <CompassAssistantProvider>
```

Comment:
CompassAssistantProvider has to wrap CompassConnections so that CompassConnections has access to it.